index.html

```diff
@@ -0,0 +1,208 @@
+            <div class="text-container">
+              <h5 class="header2">Multi-Post Stories <br> Gain+Glory</h5>
+              <ul class="text">
+                <li class="text-ruby">Ruby on Rails</li>
+                <li class="text-css">CSS</li>
+                <li class="text-java">JavaScript</li>
+                <li class="text-html">HTML</li>
+              </ul>
+              <div class="button1"><button class="button-text" type="button">See project </button></div>
+            </div>
+        </div>
+        <div  class="placeholder-container">
+            <img src="images/placeholder.svg" alt="placeholder" class="placeholder">
+            <div class="text-container">
+              <h5 class="header2">Multi-Post Stories <br> Gain+Glory</h5>
+              <ul class="text">
+                <li class="text-ruby">Ruby on Rails</li>
+                <li class="text-css">CSS</li>
+                <li class="text-java">JavaScript</li>
+                <li class="text-html">HTML</li>
+              </ul>
+              <div class="button1"><button class="button-text" type="button">See project </button></div>
+            </div>
+        </div>
+        <div  class="placeholder-container">
+          <img src="images/placeholder.svg" alt="placeholder" class="placeholder">
+          <div class="text-container">
+            <h5 class="header2">Multi-Post Stories <br> Gain+Glory</h5>
+            <ul class="text">
+              <li class="text-ruby">Ruby on Rails</li>
+              <li class="text-css">CSS</li>
+              <li class="text-java">JavaScript</li>
+              <li class="text-html">HTML</li>
+            </ul>
+            <div class="button1"><button class="button-text" type="button">See project </button></div>
+          </div>
+      </div>
+      </section>
+            <section class="about-container">
+              <div class="text-aline">
+                </div>
+                <div class="text-about">
+                  <div class="plane-container">
+                    <h3>About me</h3>
+                    <div class="plane-image">
+                      <img src="images/plane.svg" alt="plane">
+                    </div>
+                  </div>
+                  
+                    <div>
+                      <p class="h6"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hesitate to contact me.</p>
+                    </div> 
+                  
+                    <div class="button2">
+                      <button class="button-resume" type="button">Get my resume </button>
+                    </div>
+                </div>
+                <div class="plane">
+                  <img src="images/square.svg" alt="square">
+                </div>
+              </div>
+               <ul class="section6">
```

Comment:
- [ ] You have a great job in implementing the changes requested by the previous reviewer clap , However, you still need to pay attention on your `Alignment` to match the design as this is part of the requirement.

below is the screenshot showing the difference. :arrow_down: 

<table>
    <tr>
      <td>Your design </td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

 
![Student Align](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/43528594/6c6219f5-9f5f-4eac-8fbf-87a4fd422200)


</td>
      <td>

![Figma Align](https://github.com/KiengPhyl/Portfolio-setup-and-mobile-first1/assets/43528594/11153383-ebf6-42d1-a2ba-1f852d927689)


</td>
    </tr>
  </table>